### DISCUSSION TITLE
这个功能怎么用，感觉挺强大的

### DISCUSSION THREAD

**Discussion by fanyty** [wanghongenpin/proxypin#309] [Q&A]
![image](https://github.com/user-attachments/assets/3271a3e7-fb84-439b-a229-41bd2c0b508d)

**Comment by fanyty** [comment]
![image](https://github.com/user-attachments/assets/14a7127b-0229-4645-a428-e5a33ce568a5)

**Comment by wanghongenpin** [comment]
就是执行js代码
